server/api/routes/main.py

```diff
@@ -1,8 +1,20 @@
 from flask import Blueprint
+from database import *
```

Comment:
Just a personal thought, so it's up to the group, but I prefer to explicitly import everything so that I can trace where everything comes from in a given file.

Reply:
Totally missed that I had this, I agree on the explicit imports